apps/condo/domains/meter/access/Meter.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Generated by `createschema meter.Meter 'number:Text; billingAccountMeter?:Relationship:BillingAccountMeter:SET_NULL; account?:Relationship:BillingAccount:SET_NULL; property:Relationship:Property:CASCADE; unitName:Text; place?:Text; resource:Relationship:MeterResource:CASCADE;'`
+ */
+
+const { RESIDENT } = require('@condo/domains/user/constants/common')
+const { canReadMeterEntity, canManageMeterEntity } = require('../utils/accessSchema')
+const { Meter } = require('../utils/serverSchema')
+const { throwAuthenticationError } = require('@condo/domains/common/utils/apolloErrorFormatter')
+
+async function canReadMeters ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    return await canReadMeterEntity({ user })
```

Comment:
why do you need to extract it from here?

Reply:
I reused this logic in MeterReadingAccess, but ok, for better readability I will remove this function